src/app/components/aboutPage/aboutPage.component.tsx

```diff
@@ -1,248 +1,38 @@
-                            <img
-                                className="icon-small"
-                                alt="Light bulb icon"
-                                src={fact}
-                            />
-                        </div>
-                        <div className="text-container">
-                            There were over 140 spaceflight launches last year
-                        </div>
-                    </a>
-                </li>
-                <li>
-                    <a
-                        href="https://www.nasa.gov/feature/facts-and-figures"
-                        target="_blank"
-                    >
-                        <div className="icon-container">
-                            <img
-                                className="icon-small"
-                                alt="Light bulb icon"
-                                src={fact}
-                            />
-                        </div>
-                        <div className="text-container">
-                            There are at least 7 people on space station right
-                            now
-                        </div>
-                    </a>
-                </li>
-                <li>
-                    <a
-                        href="https://www.nasa.gov/feature/facts-and-figures"
-                        target="_blank"
-                    >
-                        <div className="icon-container">
-                            <img
-                                className="icon-small"
-                                alt="Light bulb icon"
-                                src={fact}
-                            />
-                        </div>
-                        <div className="text-container">
-                            In 24 hours, the space station is travelling through
-                            16 sunrises and sunsets
-                        </div>
-                    </a>
-                </li>
-                <li>
-                    <a
-                        href="https://www.nasa.gov/feature/facts-and-figures"
-                        target="_blank"
-                    >
-                        <div className="icon-container">
-                            <img
-                                className="icon-small"
-                                alt="Light bulb icon"
-                                src={fact}
-                            />
-                        </div>
-                        <div className="text-container">
-                            The space for living and working on space station is
-                            larger than a six-bedroom house
-                        </div>
-                    </a>
-                </li>
-            </ul>
-
-            <div className="about-page-item-title">
+            </section>
+            <List listItems={config.staticContent.aboutPage.facts} />
```

Comment:
List is a very generic name, does this component will be able to handle ANY list passed as listItems?

Reply:
done